### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by IlyaBeliaev**

### CODE REVIEW

File: www/package.json [hunk 1]
```diff
@@ -37,20 +37,20 @@
       }
     },
     "deploy": {
-      "command": "npm run clean && npm run compile",
+      "command": "npm --harmony run clean && npm --harmony run compile",
```
Comment (linusnorton) [inline www/package.json]: next time just install node 6

File: www/src/components/journey/form/JourneyForm.jsx [hunk 2]
```diff
@@ -31,30 +31,52 @@ export default class JourneyForm extends React.Component {
     defaultDestenition: null
   };
 
-  getJourneyUrl() {
-    return encodeURI('/journey/' + this.state.origin + '/' + this.state.destination + '/1')
+  getJourneyUrl(origin, destination) {
+    return encodeURI(`/journey/${origin}/${destination}/1`);
```
Comment (linusnorton) [inline www/src/components/journey/form/JourneyForm.jsx]: Think this should be the station code and also the date. What is the `/1` for?

File: www/src/utils/LocationsHelper.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+export default class LocationsHelper {
+  static getNameByCode(locations, code) {
+    const item = _.find(locations, x => x.code.toLowerCase() == code.toLowerCase());
```
Comment (linusnorton) [inline www/src/utils/LocationsHelper.js]: `===`

File: www/src/components/journey/form/JourneyForm.jsx (hunk 2 above)
Comment (IlyaBeliaev) [inline www/src/components/journey/form/JourneyForm.jsx]: It is a temp value. Actually, it's a date.

File: www/src/utils/LocationsHelper.js (hunk 3 above)
Comment (IlyaBeliaev) [inline www/src/utils/LocationsHelper.js]: ok

File: www/package.json (hunk 1 above)
Comment (IlyaBeliaev) [inline www/package.json]: I will try, but think that node 6 also need this argument

**Pull request merged**